source/_integrations/proximity.markdown

```diff
@@ -20,23 +20,31 @@ Some examples of its use include:
 - Increase thermostat temperature as you near home
 - Decrease temperature the further away from home you travel
 
-The Proximity entity which is created has the following values:
-
-- `state`: Distance from the monitored zone (in `unit_of_measurement`)
-- `dir_of_travel`: Direction of the closest device or person to the monitored zone. Values are:
-  - `not set`
-  - `arrived`
-  - `towards`
-  - `away_from`
-  - `unknown`
-  - `stationary`
-- `unit_of_measurement`: Measurement of distance. Values are:
-  - `km`
-  - `m`
-  - `mi`
-  - `yd`
-  - `ft`
-- `nearest`: The device or person which is nearest to the zone
+## Sensors
+
+The following sensor entities will be created
+
+### Distance
+
+For each tracked device or person, the distance from the monitored zone in an unit depending on your [Home Assistant Unit System](/docs/configuration/basic) selection.
```

Comment:
```suggestion
For each tracked device or person, the distance from the monitored zone in a unit depending on your [Home Assistant Unit System](/docs/configuration/basic) selection.
```

Reply:
It's weird to not have a verb in these sentences. Not sure what would be better.

Reply:
```suggestion
For each tracked device or person, a sensor showing the distance from the monitored zone in an unit depending on your [Home Assistant Unit System](/docs/configuration/basic) selection is created.
```

what about this one?

Reply:
Yes, that's good. But replace an with a.